docs/source/dev_guide/release.rst

```diff
@@ -3,124 +3,85 @@ How to Prepare a Release
+        git checkout -b v<version> -t origin/<version>
 
-5. Commit and push your changes.
+3. Bump version using tbump.
 
     ::
 
-        git commit -am 'Changes before release.'
-        git push <fork-origin> <branch-name>
-
-6. Create a pull request to the main repo and merge it.
+        # Exclude "v" and <version> should match step 2
+        # --no-tag is required since tagging is handled in "Releasing on GitHub"
+        $ tbump <version> --no-tag
+
+        :: Bumping from 1.0.0rc3 to 1.1.0
+        => Would patch these files
+        - setup.py:26 version="1.0.0rc3",
+        + setup.py:26 version="1.1.0",
+        - zppy/__init__.py:1 __version__ = "v1.0.0rc3"
+        + zppy/__init__.py:1 __version__ = "v1.1.0"
+        - conda/meta.yaml:2 {% set version = "1.0.0rc3" %}
+        + conda/meta.yaml:2 {% set version = "1.1.0" %}
+        - tbump.toml:5 current = "1.0.0rc3"
+        + tbump.toml:5 current = "1.1.0"
+        => Would run these git commands
+        $ git add --update
+        $ git commit --message Bump to 1.1.0
+        $ git tag --annotate --message v1.1.0 v1.1.0
+        $ git push origin main
+        $ git push origin v1.1.0
+        :: Looking good? (y/N)
+        >
+4. Create a pull request to the main repo and merge it.
 
 .. _github-release:
 
-Creating A Release On GitHub
-----------------------------
-
-1. Go to the Releases on the GitHub repo of the project
-`here <https://github.com/E3SM-Project/zppy/releases>`_.
-and draft a new release.
-
-2. ``Tag version`` and ``Release title`` should both be the version, including the "v".
-(They should match ``git_rev`` in step 4 of the previous section).
-``Target`` should be ``main``. Use ``Describe this release`` to write what features
-the release adds. You can scroll through
-`zppy commits <https://github.com/E3SM-Project/zppy/commits/main>`_ to see
-what features have been added recently.
-
-Note that you can also change the branch which you want to release from,
-this is specified after the tag (@ Target: ``main``).
-
-The title of a release is often the same as the tag, but you can set it to whatever you want.
+Releasing on GitHub
+-------------------
 
-Remember to write a description.
+1. Draft a new release `here <https://github.com/E3SM-Project/e3sm_diags/releases>`_.
+2. Set `Tag version` to ``v<version>``, **including the "v"**. `@Target` should be ``master``.
+3. Set `Release title` to ``v<version>``, **including the "v"**.
+4. Use `Describe this release` to summarize the changelog.
 
-3. Click "Publish release".
+   * You can scroll through `zppy commits <https://github.com/E3SM-Project/zppy/commits/main>`_ for a list of changes.
 
-Releasing The Software On Anaconda
-----------------------------------
+5. If this version is a release candidate (``<version>`` appended with ``rc``), checkmark `This is a pre-release`.
+6. Click `Publish release`.
+7. CI/CD release workflow is automatically triggered.
```

Comment:
Decreased verbiage to make the steps more concise.

I also removed the line break between the ordered bullets so that the lists are more compact.
To create unordered bullets under a number, you need to indent 3 spaces and have line breaks before and after the unordered bullets.